demo-react/src/components/Header.tsx

```diff
@@ -0,0 +1,140 @@
+import { useSpace } from '../hooks';
+import { Avatar, AvatarProps } from './Avatar';
+import { AvatarStack } from './AvatarStack';
+
+export const Header = () => {
+  const space = useSpace();
+  const members = space?.getMembers().map(({ profileData }) => {
+    return { name: profileData.name };
+  });
+  const self: AvatarProps = { name: space?.getSelf()?.profileData.name };
+
+  return (
+    <header
+      id="main-header"
+      className="bg-white"
+    >
+      <div className="py-4 mx-auto xs:justify-between xs:grid xs:grid-rows-2 xs:grid-cols-2 md:max-w-screen-2xl md:px-16 md:flex md:items-center">
+        <section>
+          <p className="font-semibold pl-8 md:text-2xl">Team Argo</p>
+          <p className="leading-5 pl-8">Pitch deck</p>
+        </section>
+
+        <section className="ml-auto flex group relative pr-8 xs:mt-2 md:mt-0">
+          <svg
```

Comment:
Would be probably good to put the SVG's into separate components for sake of readability?

Reply:
that's done 👍